spec/lib/openid_token_proxy/token/refresh_spec.rb

```diff
@@ -57,12 +57,17 @@ def index
 
     context 'when token was refreshed successfully' do
       it 'executes actions normally returning new tokens as headers' do
-        get :index, refresh_token: refresh_token
-        expect(response).to have_http_status :ok
-        expect(response.body).to eq 'Refresh successful'
-        expect(response.headers['X-Token']).to eq 'new access token'
-        expect(response.headers['X-Refresh-Token']).to eq 'new refresh token'
-        expect(response.headers['X-Token-Expiry-Time']).to eq refreshed_expiry_time.iso8601
+        OpenIDTokenProxy.configure_temporarily do |config|
+          block = double('block')
+          config.token_refreshment_hook = proc { |token| block.run(token) }
+          expect(block).to receive(:run).with(instance_of(OpenIDTokenProxy::Token))
+          get :index, refresh_token: refresh_token
```

Comment:
@jgorset could you use the `expect do |probe|` variant here as well? Then I think we're good to go! :+1: